find-route-around-barriers/src/main/java/com/esri/arcgismaps/sample/findroutearoundbarriers/MainActivity.kt

```diff
@@ -219,6 +219,7 @@ class MainActivity : AppCompatActivity() {
 
         // display the bottom sheet with directions when the button is clicked
         directionsButton.setOnClickListener {
+            if (directionsList.isEmpty()) return@setOnClickListener
```

Comment:
```suggestion
            if (directionsList.isEmpty()) return@setOnClickListener showError("Add stops on map to find route")
```
A message here could be helpful.